### PR TITLE
Updating readme to include instructions to opt out of telemetry.

### PR DESCRIPTION
**Pull request by mapill-msft**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-cli/pull/277)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,5 +94,12 @@ contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additio
 
 ## Data/Telemetry
 
-This project collects usage data and sends it to Microsoft to help improve our products and services. Note however that no data collection is performed when using your private builds.
+The winget.exe client is instrumented to collect usage and diagnostic (error) data and sends it to Microsoft to help improve the product. 
+
+Note however that if you build the client yourself the instrumentation will not be enabled and no data will be send to Microsoft.
```
Comment (denelon) [inline README.md:99]: I think this sentence is a bit more clear if the first three words are removed.

Comment (yao-msft) [inline README.md:99]: >send [](start = 111, length = 4)

sent?

File: README.md [hunk 2]
```diff
@@ -94,5 +94,12 @@ contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additio
...
 ## Data/Telemetry
 
-This project collects usage data and sends it to Microsoft to help improve our products and services. Note however that no data collection is performed when using your private builds.
+The winget.exe client is instrumented to collect usage and diagnostic (error) data and sends it to Microsoft to help improve the product. 
+
+Note however that if you build the client yourself the instrumentation will not be enabled and no data will be send to Microsoft.
+
+WinGet client respects machine wide privacy settings and users can opt-out on their device, as documented in the Microsoft Windows privacy statement [here] (https://support.microsoft.com/en-us/help/4468236/diagnostics-feedback-and-privacy-in-windows-10-microsoft-privacy).
+
+In short to opt-out, Go to Start , then select Settings  > Privacy > Diagnostics & feedback and select Basic.
```
Comment (yao-msft) [inline README.md:103]: >  [](start = 32, length = 1)

nit: extra space

Comment (yao-msft) [inline README.md:103]: >  [](start = 56, length = 1)

extra space

**Pull request merged**